letterparser/parse.py

```diff
@@ -1,4 +1,38 @@
-import pypandoc
+import os
+import docker
+from letterparser.utils import unicode_encode, unicode_decode
 
-def dummy():
-    return True
+
+DOCKER_IMAGE = "knsit/pandoc:v2.7"
+
+
+def get_docker_client():
+    return docker.from_env()
+
+
+def get_docker_volume_and_file(file_name):
+    parts = file_name.split(os.sep)
+    file_name_path = os.sep.join(parts[0:-1])
+    file_name_file = parts[-1]
+    return file_name_path, file_name_file
```

Comment:
I don't see how this relates to docker?

Reply:
I extracted it for the docker volumes value, so docker could easily reach the host's drive, but the logic does not necessarily related to docker, no. An incorrect name, I will refactor!